app-backend/api/src/main/scala/datasource/Routes.scala

```diff
@@ -153,4 +160,22 @@ trait DatasourceRoutes extends Authentication
         }
       }
     }
+
+  def listUserDatasourceActions(datasourceId: UUID): Route = authenticate { user =>
+    onSuccess(
+      DatasourceDao.getDatasourceById(datasourceId, user).transact(xa).unsafeToFuture
+    ) { datasourceO =>
+      datasourceO match {
+        case Some(datasource) =>
+          if (user.isSuperuser || datasource.owner == user.id) {
+            complete(List("*"))
```

Comment:
It might make sense to make this `ALL` instead `*` similar to what we have on the `subject_type` field

Reply:
nevermind